adapter/stackdriver/config/config.proto

```diff
@@ -62,6 +62,90 @@ message Params {
     message MetricInfo {
         google.api.MetricDescriptor.MetricKind kind = 1;
         google.api.MetricDescriptor.ValueType value = 2;
+
+        // Describes buckets for DISTRIBUTION valued metrics.
+        // TODO: look into using google.api.distribution.proto instead - we would pull it in but it has a ton of
+        // dependencies it doesn't actually use.
```

Comment:
non-blocking: should we file an issue to track this?

Reply:
https://github.com/istio/mixer/issues/1241